README.md

```diff
@@ -40,8 +42,6 @@ Issues can be assigned to people who are members of the [ARIA Contributors](http
 When preparing GitHub pull requests:
 
 * Provide a complete summary and description for each pull request. The Working Group needs to understand the rationale for proposed changes. This description may need to be very detailed in some cases, or may be quite brief, for example if providing a change to address a spelling issue.
-* Keep pull requests specific to individual comments. Some comments may require changes to multiple source files, for example if an external link is incorrect in multiple files, and this is appropriate if the changes all relate to the same comment. However, if several separate comments are submitted together within a single pull request,  it is more difficult for the working group to parse the different points made in the comment and unless the group agrees with all aspects of the comment the pull request may need to be rejected.
```

Comment:
I don't really know what this is referring to, so I removed it. "Individual comments"?

Reply:
Maybe it was supposed to be "individual issues"? It mostly makes sense with /comment/issue/g but I also think it's fine to remove from the template. The more that appears in the template, the more unlikely people are to read it.

Reply:
This isn't actually from the PR template, its from the README!

Reply:
The more that appears in the *, the more unlikely people are to read *. :-)

Reply:
Maybe this came from wide review comments. I agree with dropping it.